steps/mentor-calibre-lvs/configure.yml

```diff
@@ -71,6 +71,8 @@ parameters:
   #
   lvs_verify_netlist: 1
 
+  lvs_adk: "inputs/adk"
+
```

Comment:
This isn't how I would approach this change.

Both "inputs" and "adk" are special to mflowgen. The "inputs" directory is one of the few restrictions that mflowgen places on structure, so we need that. The "adk" is actually listed in the inputs of this node, so we cannot change that either.

See below for my recommendation...